service/mock/service.go

```diff
@@ -44,9 +44,9 @@ type KolideService struct {
 	CheckHealthFuncInvoked bool
 }
 
-func (s *KolideService) RequestEnrollment(ctx context.Context, enrollSecret string, hostIdentifier string) (string, bool, error) {
+func (s *KolideService) RequestEnrollment(ctx context.Context, enrollSecret string, hostIdentifier string, details service.EnrollmentDetails) (string, bool, error) {
 	s.RequestEnrollmentFuncInvoked = true
-	return s.RequestEnrollmentFunc(ctx, enrollSecret, hostIdentifier)
+	return s.RequestEnrollmentFunc(ctx, enrollSecret, hostIdentifier, details)
```

Comment:
if `enrollSecret` gets its whole name in the signature, i think we should do the same for `enrollmentDetails`